src/App.tsx

```diff
@@ -1,26 +1,279 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useRef, useState } from 'react';
+import { addTodo, delTodo, get } from './api/todos';
+import { TodoList } from './components/todoList/todoList';
+import { TodoInterface } from './types/Todo';
+import { FilteredTodoList } from './components/footer/filteredTodoList';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  //#region: variables **********
+  const [todos, setTodos] = useState<TodoInterface[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [value, setValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<TodoInterface | null>(null);
+
+  // To add class 'is-active' for all todo.completed before delete
+  const [applyDeleteTodos, setApplyDeleteTodos] = useState(false);
+
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const inputForFocusRef = useRef<HTMLInputElement>(null);
+
+  // To add class 'hidden' for "ErrorNotification"
+  const notificationRef = useRef<HTMLDivElement>(null);
+
+  //#endregion: variables **********
+
+  //#region: additional functions **********
+
+  const hideNotification = () => {
+    if (notificationRef.current) {
+      notificationRef.current.classList.add('hidden');
+    }
+  };
+
+  function errorHandling(error: Error) {
```

Comment:
It is important to follow the same way of creating functions everywhere, fix it everywhere
```suggestion
  const errorHandling = (error: Error) => {
```